lib/widgets/twake_components/twake_smart_refresher.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+import 'package:pull_to_refresh/pull_to_refresh.dart';
+
+class TwakeSmartRefresher extends StatelessWidget {
+  final RefreshController controller;
+  final VoidCallback onRefresh;
+  final VoidCallback onLoading;
+  final Widget? child;
+
+  const TwakeSmartRefresher({
+    Key? key,
+    required this.controller,
+    required this.onRefresh,
+    required this.onLoading,
+    this.child,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SmartRefresher(
+      enablePullUp: true,
```

Comment:
IMO: Not set default is `true`. I think it's optional

Reply:
done